src/app/personsok/PersonsokSkjemaElementer.tsx

```diff
@@ -37,28 +40,12 @@ const InputLinje = styled.div`
     }
 `;
 
-const DatovelgerStyle = styled.div`
-    flex-direction: column;
-    padding-right: 0.5em;
-    margin-bottom: 1em;
-    font-family: 'Source Sans Pro', Arial, sans-serif;
-    font-size: 1rem;
-    line-height: 1.375rem;
-    font-weight: 400;
-`;
-
-const DatolabelStyle = styled.div`
-    margin-bottom: 0.5em;
-`;
-
 function PersonsokSkjemaElementer(props: { form: PersonsokSkjemaProps }) {
-    const [tilAvgrensing, settTilAvgrensing] = useState<DatovelgerAvgrensninger | undefined>(undefined);
-    const datoChanger = (dato?: string) => {
-        props.form.actions.settFodselsdatoFra(dato);
-        settTilAvgrensing({ minDato: dato });
-    };
     return (
         <FormStyle>
+            <TittelStyle>
+                <Innholdstittel>Avansert Søk</Innholdstittel>
```

Comment:
Om `TittelStyle` bare står for litt luft under så kan det erstattes med `<Innholdstittel className="blokk-xl" />

Reply:
Eventuellt `const TittelStyle = styled(Innholdstittel)`